engines/agi/text.cpp

```diff
@@ -563,11 +583,17 @@ void TextMgr::statusDraw() {
 		clearLine(_statusRow, 15);
 
 		charAttrib_Set(0, 15);
-		charPos_Set(_statusRow, 1);
+		if (!_vm->isLanguageRTL())
+			charPos_Set(_statusRow, 1);
+		else
+			charPos_Set(_statusRow, 21);
```

Comment:
Operator `?:`, and avoid magic numbers.

Reply:
Again, I disagree with the ternary.
But I'm fixing that magic 21.